Lib/test/test_unicode_file_functions.py

```diff
@@ -104,7 +104,7 @@ def test_failures(self):
             self._apply_failure(os.listdir, name)
 
     if sys.platform == 'win32':
-        # Windows is lunatic. Issue #13366.
```

Comment:
The perjorative dig should just go.  Replace with 'Two exceptions are possible.'